static/src/javascripts/bootstraps/liveblog.js

```diff
@@ -102,8 +102,9 @@ define([
                     eventId = $el.attr('data-event-id'),
                     title = $('.timeline__title', $el).text(),
                     targetEl = qwery('#' + eventId),
-                    dim = bonzo(targetEl).offset();
-                scroller.scrollTo(dim.top - 12, 500, 'easeOutQuint');
+                    dim = bonzo(targetEl).offset(),
+                    scrollAmount = config.switches.viewability ? dim.top - 52 : dim.top - 12;
```

Comment:
Can we put `52`, `12` and `500` in a variable or maybe in `config`. It's hard to understand magic numbers. Why that exact value?